packages/api/cms-api/src/sentry/sentry.module.ts

```diff
@@ -0,0 +1,47 @@
+import { DynamicModule, Module } from "@nestjs/common";
+import { APP_INTERCEPTOR } from "@nestjs/core";
+
+import { SENTRY_CONFIG } from "./sentry.constants";
+import { SentryInterceptor } from "./sentry.interceptor";
+
+type SentryNodeOptions = Omit<import("@sentry/node").NodeOptions, "dsn" | "environment"> & {
```

Comment:
instead you can use `import type { NodeOptions } from "@sentry/node";` to only import types

Reply:
done